apps/src/templates/RegionalPartnerSearch.jsx

```diff
@@ -167,13 +180,17 @@ class RegionalPartnerSearch extends Component {
           <div>
             <br />
             <div>
-              We are unable to find this ZIP code. You can still apply directly:
+              We are unable to find this ZIP code.
+              {!this.state.applicationsClosed &&
+                ' You can still apply directly:'}
             </div>
-            <StartApplicationButton
-              buttonOnly={true}
-              nominated={this.state.nominated}
-              priorityDeadlineDate={appsPriorityDeadlineDate}
-            />
+            {!this.state.applicationsClosed && (
+              <StartApplicationButton
+                buttonOnly={true}
+                nominated={this.state.nominated}
+                priorityDeadlineDate={appsPriorityDeadlineDate}
+              />
+            )}
```

Comment:
it looks like while the ajax request is pending, or if it fails, we'll show the StartApplicationButton, even if applications are closed. Is that correct? If so, I would suggest a solution which never indicates that applications are open when they are actually closed. one idea would be to invert the api logic to be `applications_open` instead.

Reply:
good catch! changed